presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -169,19 +172,57 @@ See also :doc:`/language/timestamp`
 
     Example: ``TIME '01:02:03.456 America/Los_Angeles'``
 
+.. _timestamp-data-type:
+
 ``TIMESTAMP``
 ^^^^^^^^^^^^^
 
     Instant in time that includes the date and time of day without a time zone.
     Values of this type are parsed and rendered in the session time zone.
+    Equivalent to ``TIMESTAMP WITHOUT TIME ZONE``, ``TIMESTAMP(3)``, and
+    ``TIMESTAMP(3) WITHOUT TIME ZONE``.
 
     Example: ``TIMESTAMP '2001-08-22 03:04:05.321'``
 
+``TIMESTAMP(P) WITHOUT TIME ZONE``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    Instant in time that includes the date and time of day without a time zone
+    with ``P`` digits of precision for the fraction of seconds.
+
+    You can use ``TIMESTAMP(P)`` as an equivalent shorter name. The default
+    ``TIMESTAMP`` is an alias for ``TIMESTAMP(3)`` and means that up to
+    millisecond values are included. A precision of up to 12, so picoseconds, is
+    supported.
+
+    You can construct a timestamp value with the TIMESTAMP literal expression.
+    Alternatively, language constructs such as ``localtimestamp(p)``, or a
+    number of :doc:`date and time function and operators </functions/datetime>`
+    can return timestamp values.
+
+    Casting to lower precision causes rounding, and to higher precision causes
+    appended zero values.
```

Comment:
maybe explicit: "causes rounding (not truncation)"

the other part: values -> digits; but i would rather omit this.

Reply:
Lets leave it in to be explicit.